elements/lisk-p2p/src/utils/validate.ts

```diff
@@ -180,20 +185,25 @@ export const validateRPCRequest = (request: unknown): P2PRequestPacket => {
 		throw new InvalidRPCRequestError('Request procedure name is not a string');
 	}
 
+	if (rpcRequest.data !== undefined && typeof rpcRequest.data !== 'string') {
+		throw new InvalidRPCRequestError('Request data is not a string or undefined');
+	}
+
 	return rpcRequest;
 };
 
-export const validateProtocolMessage = (message: unknown): P2PMessagePacket => {
+export const validateProtocolMessage = (message: P2PMessagePacket): void => {
```

Comment:
message comes from the network and in order to avoid assuming any type, the type was set to `unknown`, so i think we should keep it `unknown`